rootfs/etc/nginx/conf.d/default.template

```diff
@@ -1,7 +1,7 @@
 vhost_traffic_status_zone;
 
 gzip on;
-gzip_types text/html text/plain application/json;
+gzip_types text/plain application/json;
```

Comment:
Is this removed because it's already included somewhere else?

Reply:
It's there by default.

Reply:
read the description of the PR :)

Reply:
Oops, i've missed the description, sorry :)